platform/binance/api.go

```diff
@@ -230,3 +249,16 @@ func decimalPlaces(v string) int {
 	}
 	return len(s[1])
 }
+
+// checkNotZeroValue check that string value is not 0
+func checkNotZeroValue(value string) (bool, error) {
+	valueFloat, err := strconv.ParseFloat(value, 64)
+	if err != nil {
+		return false, err
+	}
+	if valueFloat == 0 {
+		return false, nil
+	} else {
```

Comment:
you don't need this `} else {`

Reply:
basically we need it here. Because If the value will be != 0  like ("0.1') without else we will get false - what means that result of  ("0.1" is not 0) is false => 0.1 == 0, so we need it.

I can change the function to reverse and check that value is 0  `checkZeroValue(value string)` and return true if it is zero, 
but basically it is absolutely the same thing

Reply:
Done